simplq/src/components/common/Footer.jsx

```diff
@@ -1,86 +1,84 @@
-import React from 'react'
-import styles from './../../styles/footer.module.scss';
+import React from 'react';
+import styles from '../../styles/footer.module.scss';
 import { ForkOnGithubButton } from './Button/Button.stories';
 
-
-export const Footer = () => (
-    <div className={styles['footer']}>
-      <div className={styles['main-card-container']}>
-        <div className={styles['card']}>
-          <div className={styles['card-header']}>
-            <div className={styles['simpleQ-image-container']}>
-              <img
-                src="/images/new_logo.svg"
-                className={styles['new-logo-image']}
-                alt="simplQ new logo"
-              />
-            </div>
-            <div className={styles['logo-header-container']}>
-              <h1>SimplQ</h1>
-            </div>
+export default () => (
+  <div className={styles['footer']}>
+    <div className={styles['main-card-container']}>
```

Comment:
Just made it the default export. @daltonfury42, I feel that the footer font is way too big to fit in with the rest of the site. What do you think about reducing it?

Reply:
Let's leave it for now and tweak at the end, but yeah, we should reduce it.